src/VisualStudio/Core/Impl/Options/AbstractOptionPageControl.cs

```diff
@@ -177,4 +188,19 @@ public class RadioButtonCheckedConverter : IValueConverter
             return value.Equals(true) ? parameter : Binding.DoNothing;
         }
     }
+
+    public class CheckBoxCheckedToIntConverter : IValueConverter
```

Comment:
Do I dare ask what this is?

Reply:
I should have commented. I'll add a comment.

Reply:
This was to keep compatibility with the setting used in the Preview Features page. That value is an integer where -1 was disabled, 0 was use the flight information, and 1 was enabled.